### PR TITLE
Problem: ibc relayer don't work well with default tx prioritization strategy

### PR DESCRIPTION
**Pull request by yihuang**
Closes: #634
Solution:
- set dynamic fee extension option in hermes

- [x] Verify in the integration test that the extension option is indeed in effect.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: nix/sources.json [hunk 1]
```diff
@@ -72,6 +72,18 @@
         "url": "https://github.com/crypto-org-chain/gravity-bridge/archive/3d5bdc8af5227588d04064132335932ee88f57c3.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "ibc-rs": {
+        "branch": "feemarket",
+        "description": "IBC Relayer (Hermes) and Modules in Rust",
+        "homepage": "",
+        "owner": "yihuang",
+        "repo": "ibc-rs",
+        "rev": "e33cceaa247d5eda7650c24bd3968dea6a05248a",
```
Comment (yihuang) [inline nix/sources.json:81]: https://github.com/informalsystems/ibc-rs/pull/2573

File: integration_tests/configs/ibc.jsonnet [hunk 2]
```diff
@@ -5,6 +5,9 @@ config {
     'account-prefix': 'crc',
     'coin-type': 60,
     key_name: 'signer1',
+    'app-config'+: {
+      'index-events': ['ethereum_tx.ethereumTxHash', 'message.action'],
```
Comment (mmsqe) [inline integration_tests/configs/ibc.jsonnet:9]: could append like `super['index-events'] + ['message.action']`

Comment (yihuang) [inline integration_tests/configs/ibc.jsonnet:9]:
```suggestion
      'index-events': super['index-events'] + ['message.action'],
```

File: integration_tests/cosmoscli.py [hunk 3]
```diff
@@ -189,6 +189,12 @@ def query_all_txs(self, addr):
         )
         return json.loads(txs)
 
+    def txs(self, events: str):
```
Comment (thomas-nguy) [inline integration_tests/cosmoscli.py:192]:
```suggestion
    def txs_search(self, events: str):
```

Comment (yihuang) [reply integration_tests/cosmoscli.py]: renamed to `tx_search`.

File: integration_tests/configs/ibc.jsonnet [hunk 4]
```diff
@@ -128,9 +131,13 @@ config {
           },
         },
         gas_price: {
-          price: 10000000000000,
+          price: 10000000000000000,
```
Comment (thomas-nguy) [inline integration_tests/configs/ibc.jsonnet:134]: btw why does it set extremely high? it takes the minimum by default?

**Review by thomas-nguy — APPROVED** [review]
lgtm otherwise

File: integration_tests/configs/ibc.jsonnet (hunk 4 above)
Comment (yihuang) [reply integration_tests/configs/ibc.jsonnet:134]: it's to emphasis that the `max_priority_price` take effects.

**Pull request merged**